pkg/platform/model/inventory.go

```diff
@@ -28,11 +29,22 @@ var (
 )
 
 // IngredientAndVersion is a sane version of whatever the hell it is go-swagger thinks it's doing
-type IngredientAndVersion = inventory_models.V1IngredientAndVersion
+type IngredientAndVersion struct {
+	*inventory_models.V1IngredientAndVersion
+	Namespace string
+}
 
-// Platforms is a sane version of whatever the hell it is go-swagger thinks it's doing
+// Platform is a sane version of whatever the hell it is go-swagger thinks it's doing
 type Platform = inventory_models.V1Platform
 
+const (
+	// PackageNamespacePrefix is the namespace prefix for packages
+	PackageNamespacePrefix = "language"
+
+	// BundlesNamespacePrefix is the namespace prefix for bundles
+	BundlesNamespacePrefix = "bundles"
+)
```

Comment:
Can we put this in http://github.com/ActiveState/cli/blob/84104c044fd08c96b63c769ab9b657879e26901a/pkg/platform/model/vcs.go#L1-L1

So it's nice and central